js/App.js

```diff
@@ -1,13 +1,15 @@
 const React = require("react");
 const Title = require("./components/Title");
 const Description = require("./components/Description");
+const Color = require("./components/Color");
 
 class App extends React.Component {
 	render() {
 		return (
 			<div className="app">
 				<Title>Test Title</Title>
-				<Description library="will.js"/>
+				<Description library="Nick.js"/>
+				<Color/>
```

Comment:
This component should take in a color property, and some content to be displayed.